fontforge/python.c

```diff
@@ -8269,6 +8269,7 @@ static struct flaglist import_ps_flags[] = {
     { "removeoverlap", 0 },			/* Obsolete */
     { "handle_eraser", sf_handle_eraser },
     { "correctdir",  sf_correctdir },
+    { "imagereferenceonly",  sf_imagereferenceonly },
```

Comment:
At the risk of starting the nitpicking too early, is it necessary to say "imagereferenceonly"? Would there be anything vague or misleading about "referenceonly"?